### ISSUE TITLE
Moving Repo

### ISSUE THREAD

**Issue by i8wu** [xgfe/react-native-simple-toast#9]
Hi guys,

I'm pretty sure at this point the repo seems to be abandoned for whatever reason. I'm working with NPM and we should be able to keep the react-native-simple-toast package name while moving to a different repo. I think we'll get ownership of the package early May.

~@Naoto-Ida has volunteered to host the new repo on his fork. Could you please merge in the PRs that are currently active into your fork?~
https://github.com/vonovak/react-native-simple-toast

Tagging people that may be interested...
@messense  @vonovak

**Comment by x-bingxin** [comment]
I tried the _npm install_ today, still link to this repo, is there any solutions so I can make it work temporarily?
![image](https://cloud.githubusercontent.com/assets/9410789/25796167/0de5cce0-340b-11e7-91fc-02c12a3c2dac.png)

**Comment by i8wu** [comment]
Does https://github.com/xgfe/react-native-simple-toast/pull/8 fix it? @Naoto-Ida hasn't responded yet so repo hasn't been moved.

**Comment by vonovak** [comment]
@Bing-Xia I dont think you issue is related to this. Did you link the module?

**Comment by vonovak** [comment]
@i8wu I have no problem hosting this package, I use it in production

**Comment by i8wu** [comment]
Works for me @vonovak , does your fork merge in or solve the PRs that are open in this repo?

**Comment by vonovak** [comment]
@i8wu I currently don't have a fork, but can make it happen. Oddly enough, the latest works for me even though I'm using RN 0.44. Should I go ahead? Do you have the package ownership?

**Comment by i8wu** [comment]
Yeah I have package ownership, I'm on RN0.42 and it's working too..maybe hold off on #8 ? Other 2 PRs seem fine though. Thanks.

**Comment by vonovak** [comment]
@i8wu here it goes https://github.com/vonovak/react-native-simple-toast

**Comment by i8wu** [comment]
Could you give me write access to the repo? (To make release tags)

Also in the readme it should be `react-native link` not `react-native-link` right?

**Comment by vonovak** [comment]
sure, will do. yeah, you're right, I'm probably blind today, cannot get the readme right

**Comment by i8wu** [comment]
Do you think you could merge in https://github.com/xgfe/react-native-simple-toast/pull/6
https://github.com/xgfe/react-native-simple-toast/pull/5 ? Unless you see something wrong with em. Thanks!

**Comment by vonovak** [comment]
#6 is merged in, #5 is simply mentioned in readme...

**Comment by i8wu** [comment]
Oh, ok. Should be good then. I'll tag a release.

**Comment by Naoto-Ida** [comment]
Sorry I've been out of the conversation.
I'm here now.
Anything to be looked at?

**Comment by i8wu** [comment]
It's cool, everyone's probably busy with their own thing. Could you review if your PR https://github.com/xgfe/react-native-simple-toast/pull/8 is actually needed? @vonovak and I are on RN0.40+ and toast works without that change. (I'm on Android and iOS)

**Comment by i8wu** [comment]
@vonovak could you enable issues on the fork? https://softwareengineering.stackexchange.com/questions/179468/forking-a-repo-on-github-but-allowing-new-issues-on-the-fork/179470

**Comment by vonovak** [comment]
@i8wu done

**Comment by silldepair** [comment]
undefine is not object (evaluating 'RCTToastAndroid.SHORT') pr https://github.com/xgfe/react-native-simple-toast/pull/8 didnt fix it in my case, 
what i did is 
- react-native link react-native-simple-toast (not fixed)
- change file LRDRCTSimpleToast.m import to :
#import <UIKit/UIKit.h>
#import "React/RCTBridgeModule.h"
#import "UIView+Toast.h"
 then build again also not fixed

i am using rn 0.52.3
